.circleci/images/basic-docker-base/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:17.10
```

Comment:
Can we either use 16.04 or 18.04

Reply:
@JasoonS should be a quick change :)

Reply:
Yes, this code is a piece of history ;)

I think the idea from the retreat was to use the `admin-cli` (perhaps it needs a better name) base image as the base image for all our tests.

So I decided to delete this file now, the admin-cli image will be kept up to date instead, and has all the docker requirements.

Side note:

This will be extremely useful for our tests (the others much more than this one), and also debugging our circleci tests by using the `re-run with ssh` option in circleci. Then you can ssh into a machine with the cli tool already installed to help you work out what is wrong.

Reply:
Okay great!